### ISSUE TITLE
Submitting a CZI EOSS proposal for Jupyter accessibility practices

### ISSUE THREAD

**Issue by isabela-pf** [jupyter/accessibility#36]
Hello everyone! 

@trallard, @tonyfast, and I have been working on applying for a cycle 4 [CZI EOSS grant](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/). I've mentioned this in some of the past JupyterLab accessibility meetings, but I wanted to make sure to call it out here so more people are aware. 

This application covers the following work:
- Building a testing strategy comprised of both manual and automated continuous integration testing
- Addressing WCAG violations in JupyterLab and other core Jupyter ecosystem projects it relies on (such as Lumino or the PyData Sphinx documentation theme).
- Updating documentation on best practices for development, documentation, design, and community guidelines to include accessibility as a critical part of the contribution process. 

The idea is to continue and complete the goals we've already started with JupyterLab and then extend the knowledge and practices we develop from that experience to elsewhere in the Jupyter ecosystem. Even though we are planning to start with JupyterLab, I don't want this to be work that only benefits the one project.

Here's my spot to tag people who probably want this proposal update (let me know if you think of anyone else I should tag): @jasongrout @choldgraf @willingc @saulshanabrook

**Comment by choldgraf** [comment]
I think this is a great idea - thanks for leading this effort @isabela-pf @trallard and @tonyfast. I am happy to help. I think that @bollwyvl might be interested too - he has given helpful advice about a11y on the PyData Sphinx Theme!

**Comment by jasongrout** [comment]
Is there a central place for people putting in Jupyter CZI grants to coordinate? We've also been thinking about putting in a proposal for a community events manager. It would be nice if we could coordinate the various proposals going in, at the very least so we aren't stepping on each other's toes.

**Comment by trallard** [comment]
I wonder if it would be best to place them in a Discourse discussion? rather than having them in per-project repos?

**Comment by meeseeksmachine** [comment]
This issue has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/coordinating-czi-eoss-applications-across-the-community/8472/1

**Comment by trallard** [comment]
As for community management @jasongrout  I know Numpy, Pandas, Matplotlib and SciPy are working on a joint proposal through the DEI application (with a focus on onboarding a community manager). Might be worth syncing with them and join forces?

cc @ralfgommers who is working on this

**Comment by choldgraf** [comment]
Good point about coordinating a bit @trallard - I just opened up a community forum issue to see if there were others who are thinking of applying for something:

https://discourse.jupyter.org/t/coordinating-czi-eoss-applications-across-the-community/8472/1

**Comment by rgommers** [comment]
> As for community management @jasongrout I know Numpy, Pandas, Matplotlib and SciPy are working on a joint proposal through the DEI application (with a focus on onboarding a community manager). Might be worth syncing with them and join forces?

We are indeed. Happy to chat about it, but I'm not sure we should enlarge the group of projects, especially because there's a bit less synergy with Jupyter than between NumPy-Pandas-Matplotlib-SciPy. We're already struggling to scope it well and keep it below the max budget, four projects is a lot to cover.

**Comment by choldgraf** [comment]
Conversely - I'd also argue that Jupyter itself is a complex-enough ecosystem that it warrants its own community manager and strategist. This might be something that we could synergize with parts of the JupyterHub DEI submission? I think our hope for that one was that we could pay for ~50% of a person's time to do a combination of:

- Strategic planning around community growth
- Wrangling mentorship and advising for Outreachy interns (or other similar projects)
- General community-facing work (with a focus on growth within the developer community, and creating pathways between the user and developer community)

Basically, somebody that could take the [Mozilla Open Leadership Framework](https://mozilla.github.io/open-leadership-framework/framework/) and start to apply it to the JupyterHub repositories. I think this could certainly be of value to the broader Jupyter ecosystem if others are interested!

**Comment by willingc** [comment]
Looping back to the original issue of accessibility, what needs to still be done to submit accessibility as a project?

**Comment by choldgraf** [comment]
I think mostly just supporting @trallard and @isabela-pf and @echarles in putting together the proposal.

**Comment by trallard** [comment]
Thanks for all the interest in supporting folks - agree atm we need 👀  eyes on the letter of intent so if anyone can spare any cycles it would be most appreciated

**Comment by isabela-pf** [comment]
Two quick updates on this topic:
1. I have the letter of intent submitted as a PR in #44. Check it out if you'd like!
2. We will be writing a full application for this in the next month. More updates on that to come as we just found out.

**Comment by choldgraf** [comment]
Congrats y'all! That is great news!

**Comment by isabela-pf** [comment]
As promised, here is a public draft of our application. Now's the time for feedback! [Check it out here](https://docs.google.com/document/d/1HxoZTDKuHz58VEPpHYJDloMfHFkbxCJNKOu84OYvylI/edit?usp=sharing).

For those that don't know, this application is due by May 19th, so feedback by Monday the 17th at the latest would be helpful to give us a little time to address it. Let me know here if you have any other questions, and thanks in advance for giving this application a review.

Tagging some people I know are interested (let me know if there's others I should add): @willingc @choldgraf @telamonian @captainsafia @MSeal

**Comment by fperez** [comment]
Thx @isabela-pf for that draft link, very useful! I linked it in the [governance discussion](https://github.com/jupyter/governance/issues/100) so anyone going to provide feedback there doesn't miss it.

**Comment by isabela-pf** [comment]
This proposal has been submitted! Thanks to everyone involved and supporting accessibility efforts in Jupyter spaces.
